implementations/rust/ockam/ockam_command/src/message/mod.rs

```diff
@@ -1,9 +1,11 @@
-mod send;
-
-use crate::HELP_TEMPLATE;
 use clap::{Args, Subcommand};
+
 use send::SendCommand;
 
+use crate::HELP_TEMPLATE;
```

Comment:
Style wise I'd collapse the spaces between these lines to make the section more concise

Reply:
That's because of the clion's default rules for rustfmt. It separates imports in different sections (local crates, external deps, etc). For some reason clion has this option enabled by default: https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#StdExternalCrate%5C%3A

Maybe we should have a `.rustfmt.toml` to ensure we all apply the same rules.

Reply:
Hm, maybe. Although I think net every file needs to have the same rules for imports. I do generally also try to separate different "types" of imports, but when there's just 2 or 3 lines it feels overkill to also add blank lines in between

Reply:
I've reverted those changes for now.